lib/Installation/DiskActivation/DASDDiskManagementPage.pm

```diff
@@ -0,0 +1,60 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles DASD Disk Management page
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::DiskActivation::DASDDiskManagementPage;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+
+sub init {
+    my ($self) = @_;
+    $self->SUPER::init();
+    $self->{tb_minimum_channel} = $self->{app}->textbox({id => 'min_chan'});
```

Comment:
iirc, we agreed `txt_` instead of `tb` in a recent meeting, we need three letters, in that way differentiates better from table `tbl`. For `menu` we need to find one :) `mec` ? `mco` just `men` ? I would just try one (your choice of 3 letter) and when more menu appear we can see if makes sense to remark 'menu' or 'collection' in the abbreviation. wdyt?

Reply:
good point